lib/js-api/test/content-services/agentsApi.spec.ts

```diff
@@ -67,5 +67,17 @@ describe('AgentsApi', () => {
                 done();
             });
         });
+
+        describe('getAgentAvatar', () => {
+            it('should get agent avatar', (done) => {
```

Comment:
This test is failing for me. Please remove caches (nx reset) before running that test so you should be able to see that error. As on below screenshot - first test passes but this one failed.  
<img width="1718" alt="image" src="https://github.com/user-attachments/assets/73624804-802b-4c80-8216-1ac137198941">